core/src/components/skeleton-text/skeleton-text.scss

```diff
@@ -46,24 +46,31 @@ span {
 
 :host(.skeleton-text-animated) {
   position: relative;
-
-  background: linear-gradient(to right, $skeleton-text-background 8%, $skeleton-text-background-animated 18%, $skeleton-text-background 33%);
-  background-size: 800px 104px;
```

Comment:
Are we able to add this back? The shimmer seems to be wider as a result of this removal.